datastore/src/main/java/org/jboss/pnc/datastore/DefaultDatastore.java

```diff
@@ -130,11 +130,14 @@ public DefaultDatastore(ArtifactRepository artifactRepository,
     @TransactionAttribute(TransactionAttributeType.REQUIRED)
     public BuildRecord storeCompletedBuild(BuildRecord.Builder buildRecordBuilder) {
         BuildRecord buildRecord = buildRecordBuilder.build();
+        logger.debug("Storing completed build {}.", buildRecord);
+
         buildRecord.setDependencies(saveArtifacts(buildRecord.getDependencies()));
         buildRecord.setBuiltArtifacts(saveArtifacts(buildRecord.getBuiltArtifacts()));
+
         logger.debug("Saving build record {}.", buildRecord);
         buildRecord = buildRecordRepository.save(buildRecord);
-        logger.trace("Build record saved {}.", buildRecord);
+        logger.debug("Build record saved {}.", buildRecord);
 
```

Comment:
Now there are 3 messages on the same level for the same operation. There should be one debug and others trace.

Reply:
Marking the second one as trace, but I'd like to leave the last one as debug, because I find it helpful, once investigating the logs from real deployments.